ocs_ci/ocs/defaults.py

```diff
@@ -35,3 +35,11 @@
 OCS_OPERATOR_NAME = "ocs-operator"
 LOCAL_STORAGE_OPERATOR_NAME = "local-storage-operator"
 LIVE_CONTENT_SOURCE = "redhat-operators"
+
+# Noobaa status output
+NOOBAA_STATUS_CONTENT_COUNT = [
```

Comment:
A more pythonic way would be to use a list of dictionaries.
```
NOOBAA_STATUS_CONTENT_COUNT = {
'CustomResourceDefinition': 5, 
... 
}
```

Reply:
Changed to dict.